openassessment/templates/openassessmentblock/peer/oa_peer_assessment.html

```diff
@@ -66,9 +66,11 @@ <h3 class="peer-assessment__display__title">
                                     </h3>
                                 </header>
 
-                                {% include "openassessmentblock/oa_submission_answer.html" with answer=peer_submission.answer answer_text_label="Your peer's response to the question above:" %}
+                                {% trans "Your peer's response to the question above" as transated_label %}
+                                {% include "openassessmentblock/oa_submission_answer.html" with answer=peer_submission.answer answer_text_label=translated_label %}
```

Comment:
Update-- looks like the h2 is being intentionally hidden, but present for screen readers. Here is the label CSS:

label {
        @extend %text-sr;
      }